File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/colorselector/RectangleSelectionListener.java
package net.kdt.pojavlaunch.colorselector;

/**
 * 彩度・明度（SV）矩形領域で選択が行われたときのリスナーインターフェース。
 */
public interface RectangleSelectionListener {
    /**
     * 彩度と明度が変更されたときに呼び出されます。
     * @param luminosity 明度（0〜1）
     * @param intensity 彩度（0〜1）
     */
    void onLuminosityIntensityChanged(float luminosity, float intensity);
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/ui/view/colorselector/RectangleSelectionListener.kt
package com.arata.yukarilauncher.ui.view.colorselector

/** 彩度・明度の矩形選択が変更された際のコールバックインターフェース。 */
interface RectangleSelectionListener {
    /** 彩度・明度が変更されたときに呼び出される。 @param luminosity 明度 @param intensity 彩度（0～1） */
    fun onLuminosityIntensityChanged(luminosity: Float, intensity: Float)
}